tests/functional/test_cluster_functions.py

```diff
@@ -42,48 +45,53 @@ def fin():
 
 @pytest.fixture(scope="module")
 def client(adcm):
-    steps.upload_bundle(adcm.api.objects, BUNDLES + "cluster_bundle")
-    steps.upload_bundle(adcm.api.objects, BUNDLES + "hostprovider_bundle")
+    steps.upload_bundle(adcm.api.objects, get_data_dir(__file__, 'hostprovider_bundle'))
```

Comment:
Please do not use upload_bundle. Rewrite this function with sdk_client_ms and upload_from_fs function. You could find api.objects inside sdk_client_ms intance to return it.